sample_packages/sample_python_package/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.9
```

Comment:
We should pin the SHA256 hash of the container here

Reply:
Do you mean the sha for the image? Updated the PR. I took the latest python3.9 sha from here: https://hub.docker.com/layers/library/python/3.9/images/sha256-edee3d665aba1d84f5344eca825d0de34b38dbf77a776cafd9df65c67e240866?context=repo&tab=vulnerabilities

Reply:
yes, thanks!